pkg/secrets/passdriver/passdriver_test.go

```diff
@@ -0,0 +1,85 @@
+package passdriver
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setup() (*Driver, error) {
+	return NewDriver("gopass")
+}
+
+func TestStoreAndLookupSecretData(t *testing.T) {
+	tstdriver, err := setup()
+	require.NoError(t, err)
+	defer tstdriver.exec(context.TODO(), nil, os.Stdout, "rm", "-r", "-f", secretsPrefix)
+
+	err = tstdriver.Store("unique_id", []byte("somedata"))
+	require.NoError(t, err)
+
+	secretData, err := tstdriver.Lookup("unique_id")
+	require.NoError(t, err)
+	require.Equal(t, secretData, []byte("somedata"))
+}
+
+func TestStoreDupID(t *testing.T) {
+	tstdriver, err := setup()
+	require.NoError(t, err)
+	defer tstdriver.exec(context.TODO(), nil, nil, "rm", "-r", "-f", secretsPrefix)
+
+	err = tstdriver.Store("unique_id", []byte("somedata"))
+	require.NoError(t, err)
+
+	err = tstdriver.Store("unique_id", []byte("somedata"))
+	require.Error(t, err)
+}
+
+func TestLookupBogus(t *testing.T) {
+	tstdriver, err := setup()
+	require.NoError(t, err)
+	defer tstdriver.exec(context.TODO(), nil, nil, "rm", "-r", "-f", secretsPrefix)
+
+	_, err = tstdriver.Lookup("bogus")
+	require.Error(t, err)
+}
+
+func TestDeleteSecretData(t *testing.T) {
+	tstdriver, err := setup()
+	require.NoError(t, err)
+	defer tstdriver.exec(context.TODO(), nil, nil, "rm", "-r", "-f", secretsPrefix)
+
+	err = tstdriver.Store("unique_id", []byte("somedata"))
+	require.NoError(t, err)
+	err = tstdriver.Delete("unique_id")
+	require.NoError(t, err)
+	data, err := tstdriver.Lookup("unique_id")
+	require.Error(t, err)
+	require.Nil(t, data)
+}
+
+func TestDeleteSecretDataNotExist(t *testing.T) {
+	tstdriver, err := setup()
+	require.NoError(t, err)
+	defer tstdriver.exec(context.TODO(), nil, nil, "rm", "-r", "-f", secretsPrefix)
+
+	err = tstdriver.Delete("bogus")
+	require.Error(t, err)
+}
+
```

Comment:
Please add a test where someone tries putting in a valid file that's outside of the scope of the key directory and that it fails.

Reply:
That test is now there for all methods.